bin/zero/runtime/Cargo.toml

```diff
@@ -112,6 +112,12 @@ gamedao-signal = { path = "../../../gamedao-protocol/signal", default-features =
 gamedao-sense = { path = "../../../gamedao-protocol/sense", default-features = false }
 gamedao-treasury = { path = "../../../gamedao-protocol/treasury", default-features = false }
 
+pallet-tangram = { path = "../../../gamedao-protocol/tangram", default-features = false }
+pallet-rmrk-core = { version = "0.0.1", default-features = false, path = "../../../rmrkl/rmrk-core" }
+pallet-uniques = { version = "0.0.1", default-features = false, path = "../../../rmrkl/uniques" }
```

Comment:
can't you use the default uniques or why do you need a local version?

Reply:
I have used RMRK from Dec 2021.
Reason is simple. They RMRK team is used latest Substrate as result I used old version which compiles well with us.
In that version RMRK team made their own copy of uniques 

![image](https://user-images.githubusercontent.com/36797566/175015397-a12d0766-fafa-4a28-9e2c-204022f9e07c.png)

Reply:
yes, they also just released a bigger update which also has contracts and deploying to parachain might be sketchy if versions do not match. have you checked out most recent zero-network for compatibility?

what is the goal now? if we want to test maybe writing tests could help, otherwise we should really sync and refocus, as time moves on and many things changes over the last weeks.